spec/spreadsheets/allocations_spreadsheet_spec.rb

```diff
@@ -9,23 +9,19 @@
     [
       User.human_attribute_name('name'),
       User.human_attribute_name('specialty'),
+      Office.human_attribute_name('head'),
       Allocation.human_attribute_name('project'),
-      Allocation.human_attribute_name('start_at'),
-      Allocation.human_attribute_name('end_at'),
-      Allocation.human_attribute_name('created_at'),
-      Allocation.human_attribute_name('updated_at'),
+      Allocation.human_attribute_name('end_at')
     ]
   end
 
-  let (:body_attributes) do
+  let(:body_attributes) do
```

Comment:
We should use explicit values in this let, I mean, you should test the end result for the office's head name instead of decorating the user and taking these attributes.